runtime/bootstrap/index.ts

```diff
@@ -242,13 +244,15 @@ export async function injectExecutionContext({
   context: Context;
   fetcher: Fetcher;
   temporaryBlobStorage: TemporaryBlobStorage;
+  syncState: SyncStateService;
```

Comment:
Should probably type this as `SyncStateService | undefined` since ideally we will know enough when setting things up that we can choose not to provide this if it's not a sync.